src/main/java/org/generationcp/breeding/manager/listmanager/ListBuilderComponent.java

```diff
@@ -1496,8 +1496,9 @@ public void viewListAction() {
 		}
 	}
 
-	public void changeToListView() {
-		if (this.listInventoryTable.isVisible()) {
```

Comment:
Hi @mderamos , I wonder why you had to change the logic. Is this.listInventoryTable.isVisible() not working?

Reply:
Hi @abatac, there's an issue in isVisible when switching from list view and germplasm view. The this.listInventoryTable.isVisible() returns false even if it should return true when the view is switched from List View to Germplasm View.